### PR TITLE
Fix for the MDSD logging issues

### PR DESCRIPTION
**Pull request by Makdaam**
```release-note
MSFT: fixing MDSD logging resourceGroup
```

### CODE REVIEW

File: pkg/startup/v7/data/common/etc/mdsd.d/mdsd.xml [hunk 1]
```diff
@@ -7,7 +7,7 @@
         <Identity tenantNameAlias="ResourceName">
             <IdentityComponent name="Region">{{ .ContainerService.Location | XMLEscape }}</IdentityComponent>
             <IdentityComponent name="SubscriptionId">{{ .ContainerService.Properties.AzProfile.SubscriptionID | XMLEscape }}</IdentityComponent>
-            <IdentityComponent name="ResourceGroupName">{{ .ContainerService.Properties.AzProfile.ResourceGroup | XMLEscape }}</IdentityComponent>
+            <IdentityComponent name="ResourceGroupName">{{ .Derived.CustomerResourceGroup .ContainerService.ID | XMLEscape }}</IdentityComponent>
```
Comment (ehashman) [inline pkg/startup/v7/data/common/etc/mdsd.d/mdsd.xml]: v7 changes shouldn't be hitting master, can you target these in a separate PR to the `release-v7` branch?

Comment (jim-minter) [reply pkg/startup/v7/data/common/etc/mdsd.d/mdsd.xml]: definitely shouldn't hit master - we need to think about whether we want to backport to v7 - let's discuss on scrum

**Comment by jim-minter** [comment]
this is going in the right direction, please see the comment about azure.ParseResourceID (or whatever it's called) in the corresponding metrics PR

File: pkg/startup/v7/data/common/etc/mdsd.d/mdsd.xml (hunk 1 above)
Comment (Makdaam) [reply pkg/startup/v7/data/common/etc/mdsd.d/mdsd.xml]: sorry, misunderstood the scope and how v7.1 patch is going to happen. The v71 explanation on the scrum explained it. I'll clean it up.

**Comment by jim-minter** [comment]
lgtm, waiting for test result

**Comment by Makdaam** [comment]
/retest 
can't resolve github.com

**Comment by ehashman** [comment]
Flake #1628 

/retest

**Pull request merged**